thorlcr/master/thmastermain.cpp

```diff
@@ -1046,7 +1046,10 @@ int main( int argc, const char *argv[]  )
                 addTimeStamp(workunit, wfid, graphName, StWhenK8sStarted);
             }
 
-            cloudJobName.appendf("%s-%s", workunit, graphName);
```

Comment:
Should be same as agentexec approach, @instanceNum should always be present, but just in case (version mismatch) fallback to <workunit>-<graphName> if not.